products/sle/main.pm

```diff
@@ -1254,6 +1254,10 @@ else {
         }
         else {
             if (get_var("BOOT_TO_SNAPSHOT") && (snapper_is_applicable())) {
+                # poo#13156 is not resolved: Add support to easily switch VERSION during a test run
+                # Disable further console and x11 tests after rollback
+                # Bring the tests back if poo#13156 is done
+                set_var('INSTALLONLY', 1);
```

Comment:
hm. ok for me but an alternative which *might* be better is to set the variable on top of the file where we are also setting other variable implications. keep in mind that the current testsuite "boot_to_snapshot" on osd sets `INSTALLONLY` already so it should be removed from there afterwards